packages/pn-pa-webapp/src/i18n.ts

```diff
@@ -17,6 +34,18 @@ void i18next
       // this customization is needed to fix pn-2838
       escape: (srt: string): string => sanitizeString(srt),
     },
+    detection: {
+      order: ['customHashDetector', 'sessionStorage'],
+      lookupQuerystring: 'lang',
+      lookupSessionStorage: 'lang',
+    },
+  })
+  .then(async () => {
+    await i18next.changeLanguage();
```

Comment:
why is i18next.changeLanguage() needed?

Reply:
I removed it because the configuration had `lng: "it"`. Without this, the `changeLanguage` function is useless.